lib/beaker/dsl/install_utils.rb

```diff
@@ -1453,6 +1453,57 @@ def build_ignore_list(opts = {})
         ignore_list << '..' unless ignore_list.include? '..'
         ignore_list
       end
+
+      # This method is a fix for PUP-2365 when a windows agent cannot download
+      # modules from the forge because it doesn't have the appropriate
+      # certificates installed to validate the https connection.
+      #
+      # This method installs the certifcates required in order to that functionality
+      # to work as expected.
+      #
+      # @param [Host] host             A host object
+      #
+      def install_forge_cert(host)
```

Comment:
A windows only method with a generic name, should indicate in the name that it is limited.